source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,81 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or sections using inline
+hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a full URL.
```

Comment:
Worth mentioning that "full URL" = external page? Or maybe just saying "external website" instead?

Reply:
I incorporated the first statement here -- Didn't go with external website because it can also be internal.